apps/teams-test-app/src/components/DialogAPIs.tsx

```diff
@@ -1,13 +1,13 @@
 /* eslint-disable @typescript-eslint/ban-types */
-import { dialog, DialogInfo, IAppWindow, ParentAppWindow } from '@microsoft/teams-js';
+import { dialog, DialogInfo, IAppWindow, ParentAppWindow, TaskInfo, tasks } from '@microsoft/teams-js';
 import React, { ReactElement } from 'react';
 
 import { ApiWithoutInput, ApiWithTextInput } from './utils';
 
 const DialogAPIs = (): ReactElement => {
   const childWindowRef = React.useRef<IAppWindow | null>(null);
   const OpenDialog = (): ReactElement =>
-    ApiWithTextInput<DialogInfo>({
+    ApiWithTextInput<DialogInfo | TaskInfo>({
```

Comment:
Are these two different? I think they should be exactly the same?

Reply:
Yes. I am thinking of using teams-v1 script situation in which there is no `DialogInfo`.